westpa_wexplore/wex_utils.so

```diff
@@ -1 +1 @@
-build/lib.linux-x86_64-2.7/wex_utils.so
\ No newline at end of file
+build/lib.linux-x86_64-3.7/wex_utils.cpython-37m-x86_64-linux-gnu.so
```

Comment:
I'm not sure if there should be a `.so` file included in the repo since this could cause compatibility issues.